src/__mocks__/util.js

```diff
@@ -0,0 +1,241 @@
+/**
+ * @module
+ * @ignore
+ */
+const fs = require('fs');
+const parse = require('csv-parse/lib/sync');
+const xml2js = require('xml2js');
+const sleep = require('sleep-promise');
+const HTTP_STATUS_CODES = require('http-status-codes');
+const jsonpath = require('jsonpath');
+const crypto = require('crypto');
+const stableStringify = require('json-stable-stringify');
+const _ = require('lodash');
+
+const { logger } = require('../logging');
+
+
+const REQUESTS_CACHE = {};
+
+
+const convertOwlGraphToJson = (graph, idParser = x => x) => {
```

Comment:
Some of the methods here seem like they were not necessary to include in the mock here?